e2e/integration/frontity-01/iframe.spec.js

```diff
@@ -1,24 +1,30 @@
 import expect from "expect";
 
-describe("Native iframe lazy-load", () => {
+// eslint-disable-next-line jest/valid-describe
+describe("Native iframe lazy-load", { browser: "!firefox" }, () => {
   beforeEach(() => {
     cy.viewport(360, 640);
     cy.visit("http://localhost:3001?frontity_name=iframe");
   });
 
-  it("native lazy-load should exist", () => {
-    return cy
-      .window()
-      .its("HTMLIFrameElement")
-      .then((HTMLIframeElement) => {
-        expect("loading" in HTMLIframeElement.prototype).toBe(true);
-      });
-  });
+  if (
+    Cypress.browser.family === "chromium" &&
+    Cypress.browser.majorVersion > 77
```

Comment:
Was watching the video and notice that the caniuse table was saying that _starting_ with version 77. So, I believe this condition should be:
```suggestion
    Cypress.browser.majorVersion >= 77
```

In any case, how would you feel that this condition should actually be condition we assert inside the `expect` call? That way we don't rely on browser versions but rather on the functionality.

Reply:
> how would you feel that this condition should actually be condition we assert inside the expect call? 

Can you show an example?

Reply:
Actually, I don't know why we are testing that the native lazy load should exist in the first place.

We should test that `loading="lazy"` is added to the `<img>` or `<iframe>` tags. The rest is up to the browser.

Reply:
Exactly my point. And I was thinking to write it in a way that avoids the browser version check. Something in the lines of:

```js
it("lazy attribute should be defined if it's supported", () => {
  const isSupported = "loading" in HTMLIframeElement.prototype;
  
  if (isSupported) {
    cy.scrollTo("topLeft");
    cy.get("iframe").should("have.attr", "loading", "lazy");
  } else {
    this.skip();
  }
});
```

Reply:
Ohh, I got it now. That's a great point and makes much more sense, thanks Cris 🙂

However, in this specific case, as the `loading="lazy"` is added even if the native lazy load doesn't exist, I think I'm going to just remove the test that checks if native lazy load exists and leave only the one that checks if `loading="lazy"` was added.